src/main/resources/assets/wakes/lang/en_us.json

```diff
@@ -24,14 +29,22 @@
   "wake_spawn_rule.wakes.only_wakes": "Only wakes",
   "wake_spawn_rule.wakes.only_splashes": "Only splashes",
   "wake_spawn_rule.wakes.disabled": "Disabled",
+  "description.wakes.spawning_rules": "Defines the wakes and splashes spawning rules.",
   "option.wakes.wakes_in_running_water": "Wakes in running water",
+  "description.wakes.wakes_in_running_water": "Should wakes spawn in running water?",
 
   "option.wakes.wave_speed": "Wave speed",
+  "description.wakes.wave_speed": "The speed which the wakes travel and despawn at.",
```

Comment:
While I do state this is wake speed I realise now it's quite a bad name for it. Technically speaking and in terms of the wave simulation it is speed but when changing the value the wave doesn't really propagate faster, only farther. I might change this to something like "Wave propagation factor" or something similar.

Reply:
Pretty sure this is one of the options I struggled with making a description for. It seemed like it did a lot of things at once. I made the descriptions based on my testing of changing the settings in-game, with one or two having some code reading alongside it.

In your defense, naming config options is surprisingly difficult.